source/fleetprovisioning/FleetProvisioning.h

```diff
@@ -30,6 +30,18 @@ namespace Aws
                      */
                     bool ProvisionDevice(std::shared_ptr<SharedCrtResourceManager> fpConnection, PlainConfig &config);
 
+                    /**
+                     * \brief Map template parameters from a JSON object specified as an escaped string
+                     *
+                     * @param params Template parameters given as JSON escaped string
+                     * @return false if failed parsing template parameters
+                     */
+                    bool MapParameters(const Aws::Crt::Optional<std::string>
+                                           params); // Function must return boolean as
```

Comment:
Can we move this comment up into the comment above so that it shows up in Doxygen? You can place it two lines below the \brief tag line